packages/integration-tests/src/shared_secret.ts

```diff
@@ -0,0 +1,18 @@
+import { shortUuid } from './short_uuid.js';
+
+/**
+ * Create a unique shared secret name
+ */
+const createAmplifySharedSecretName = (): string => {
+  return 'amplifySharedSecret' + shortUuid();
+};
+
+/**
+ * Create an environment object with a shared secret
+ */
+export const createSharedSecretEnvObject = (): Record<string, string> => {
```

Comment:
Yeah I was thinking if we should do it that way or this way. My thinking was to keep the environment object consistent in the two places where we need it and avoid accidentally calling `createAmplifySharedSecretName` multiple times and having multiple shared secrets where we intended to have just one.

Reply:
Why do we need to call this in `deployment.test.ts`?

Reply:
That is to pass the environment object in `amplifyCLI` so it is in scope for the backend definition during the hotswap tests

Reply:
Otherwise the hotswap test fails [here](https://github.com/aws-amplify/amplify-backend/pull/910/files#diff-8a61ec78bef2c35e1b01da14e2a5bf23a818e098b7d1fadee44cd1b4037242caR157)

Reply:
With your latest changes it looks like the only usage is in `data_storage_auth_with_triggers`. I would suggest we just export the function above and use it there. I think having the constant define the env var key will be sufficient for keeping things in sync